py/gc.c

```diff
@@ -693,34 +693,6 @@ void gc_dump_alloc_table(void) {
         int c = ' ';
         switch (ATB_GET_KIND(bl)) {
             case AT_FREE: c = '.'; break;
-            /* this prints out if the object is reachable from BSS or STACK (for unix only)
```

Comment:
If you keep this, it seems better to discard this PR and keep everything as-is well as it keeps debugging available not just for unix but also for mac/windows builds.